src/dsp_tools/models/connection.py

```diff
@@ -119,22 +126,28 @@ def post(self, path: str, jsondata: Optional[str] = None):
         if jsondata is None:
             if self._token is not None:
                 headers = {'Authorization': 'Bearer ' + self._token}
```

Comment:
The rest of this file, from here on, contains only refactorings: I wanted to rename the response of each request from `req` to `response`. But by doing so, I had to redo the entire indentation.